### PR TITLE
Iframe extra space removed

### PR DESCRIPTION
**Pull request by Raghav888**

### CODE REVIEW

File: documentation/document.css [hunk 1]
```diff
@@ -130,6 +131,13 @@ html {
     margin: 0;
 }
 
+#code-iframe {
```
Comment (tusharkandpal) [inline documentation/document.css:134]: Make it a class as you're using it in multiple places

**Comment by tusharkandpal** [comment]
Looks good !!

File: documentation/document.css (hunk 1 above)
Comment (Raghav888) [reply documentation/document.css:134]: As Id has more specificity , I need to use it. If I use class the properties are not getting applied

**Review by tushartiwari7 — APPROVED** [review]
I liked the iframe alternative you used.

**Comment by Raghav888** [comment]
> I liked the iframe alternative you used.

Thank you

**Pull request merged**